DBZ-5181 Specify a taskId and maxTasks to run multiple EmbeddedEngines

vrajat opened the pull request:
https://issues.redhat.com/browse/DBZ-5181

A Debezium Server implementation may run multiple engines to parallelize
work between them. To enable multi-tasking an engine should be assigned
an ID and the provide total number of tasks. Using this information, it
can identify the slice of work from the source connector.

vrajat commented:
@jpechane I've updated the diff with DebeziumServer changes. Please take a look.

axeh commented:
Any update on this? It's a cool feature to have

jcechace commented:
@vrajat  Thanks for contributing. Since this PR seems to be stale I rebased it, fixed few details and opened https://github.com/debezium/debezium/pull/3989.

vrajat commented:
Thanks Jakub. I’ll focus on getting it ready for a new round of reviews

On Mon, 24 Oct 2022 at 14:14, Jakub Cechacek ***@***.***>
wrote:

> @vrajat <https://github.com/vrajat> Thanks for contributing. Since this
> PR seems to be stale I rebased it, fixed few details and opened #3989
> <https://github.com/debezium/debezium/pull/3989>.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/debezium/debezium/pull/3654#issuecomment-1288663313>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAMP7GVLXK4UCE6C45MISYLWEZEABANCNFSM5Z55HNTA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
